zenodo/modules/frontpage/templates/zenodo_frontpage/index.html

```diff
@@ -44,7 +44,7 @@
         {%- set publication_date = r.publication_date|from_isodate -%}
         {%- set creation_date = r._created|from_isodatetime -%}
         {%- set embargo_date = r.embargo_date|from_isodate -%}
-        {%- set object_type = r.resource_type|objecttype %}
+        {%- set object_type = r.resource_type.to_dict()|objecttype %}
```

Comment:
Do you know why this one is needed? It's probably better to fix in the ``objecttype`` filter or on ``ObjectType`` model.